dagshub/upload/wrapper.py

```diff
@@ -288,14 +436,38 @@ def get_file(file: Union[str, IOBase], path=None):
             raise e
 
     def _reset_dataset(self):
+        """
+        The _reset_dataset function clears the files attribute of a Dataset object.
+      
+        :return: The empty list    
```

Comment:
Please go over the returns. When there is no `return` statement, the function doesn't return anything
```suggestion
        :return: Nothing
```